### DISCUSSION TITLE
WYSIWYG images load with dynamic size

### DISCUSSION THREAD

**Discussion by PeterEhses** [directus/directus#2796] [Archive]
When I saw the image size definition option in the wysiwyg image insertion dialog I was expecting it to use an /assets/id?size url but it seems to only link to the /originals and add the size information inline. This seems like a waste of user resources since it will load the high res image file and doesn't pass the image hash for me to inject the more suitable assets url clientside. 
Correct me if I'm wrong but the only workaround that really isn't one is adding a files field and having the image information separately, which ofc. doesn't allow it to be inserted mid-body. 
Alternatively, a configurable resize-on-upload functionality would presumably work similarly well but would change the original which isn't ideal if avoidable.

**Comment by maltejur** [comment]
I also don't understand why this isn't done. The API has this capability and the UI is already there, it just has to be connected.

**Reply by benhaynes** [reply]
Time. ⏱️
